examples/cpp/helloworld/README.md

```diff
@@ -255,6 +255,18 @@ main loop in `HandleRpcs` to query the queue.
 
 For a working example, refer to [greeter_async_server.cc](greeter_async_server.cc).
 
+#### XDS client and server
```

Comment:
I don't think we actually need to document anything xds-specific here.  The only thing we're adding here is the `--target` flag, and that could be used to set the target to any URI, even one that has a scheme other than "xds".

All we really need to document is the fact that the `--target` flag can be used if the user wants to point the client at a target other than the default one.